openmc/deplete/results_list.py

```diff
@@ -192,3 +193,72 @@ def get_depletion_time(self):
         for ix, res in enumerate(items):
             times[ix] = res.proc_time
         return times
+
+    def export_to_materials_xml(self, burnup_index, nuc_with_data=None, new_xml_name=None):
+        """Export Materials.xml file from the depletion results
+
+        Parameters
+        ----------
+        burn_index : int
+            Index of burnup step to evaluate
+        nuc_with_data : Iterable of str, optional
+            nuclides list to evaluate with neutron data. If not provided,
+            nuclides from the cross_sections element of materials.xml will
+            be used. If that element is not present, nuclides from
+            OPENMC_CROSS_SECTIONS will be used.
+        new_xml_name : str, optional
+            name of xml file to put depleted materials into. If not
+            provided, no new xml file will be written.
+
+        Returns
+        -------
+        mat_file : Materials
+            A modified Materials instance containing depleted material data,
+            plus original isotopic compositions of non-depletable materials
+        """
+        result = self[burnup_index]
+
+        # Only materials found in the original materials.xml file will be
+        # updated. If for some reason you have modified OpenMC to produce
+        # new materials as depletion takes place, this method will not
+        # work as expected and leave out that material.
```

Comment:
I think this deserves to be in the docstring. It doesn't really help a developer, and we don't tell the user that we are pulling from `materials.xml` which should exist in the working directory

Reply:
Agreed, done.